.github/workflows/build.yml

```
@@ -21,6 +21,11 @@ name: build
 
 on: [push, pull_request]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
```

Please change this line to only cancel previous PR actions.

```suggestion
  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
```

---

Probably this is better.

```suggestion
  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
```

---

> Probably this is better.

I realized we need CI history of release branches or dev branches for big tasks.
So let's keep it as `${{ github.event_name == 'pull_request' }}`